src/ra.hrl

```diff
@@ -187,6 +188,17 @@
         {query_index :: integer(),
          term :: ra_term()}).
 
+-record(info_rpc,
+        {from :: ra_server_id(),
+         term :: ra_term(),
+         keys :: [ra_server:ra_server_info_key()]}).
+
+-record(info_reply,
+        {from :: ra_server_id(),
+         term :: ra_term(),
+         keys :: [ra_server:ra_server_info_key()],
+         info :: ra_server:ra_server_info() | undefined}).
```

Comment:
default this to `#{}` and remove the `undefined` option?

Reply:
Done